### PR TITLE
Enable use in TB 68+

### PR DESCRIPTION
**Pull request by neilmayhew**
This uses patches provided by the community in #577 and the dependabot fixes from #588.

---
This change is [Reviewable](https://reviewable.io/reviews/adam-p/markdown-here/592)

### CODE REVIEW

**Comment by NyanKiyoshi** [comment]
@adam-p any chance to get this reviewed? Would be great!

File: src/manifest.json [hunk 1]
```diff
@@ -46,7 +46,9 @@
 
   ,"applications": {
     "gecko": {
-      "id": "markdown-here-webext@adam.pritchard"
+      "id": "markdown-here-webext@adam.pritchard",
+      "strict_min_version": "68.0"
     }
   }
```
Comment (microlith57) [inline src/manifest.json:52]:
```suggestion
  }
```

**Comment by suchat932** [comment]
👍👍👍

**Comment by neilmayhew** [comment]
See [Markdown Here Revival](https://addons.thunderbird.net/EN-US/thunderbird/addon/markdown-here-revival/) instead.

**Comment by suchat932** [comment]
เรียบร้อย

**Pull request closed without merging**